src/appleseed/renderer/modeling/frame/frame.cpp

```diff
@@ -1010,40 +992,43 @@ namespace
         }
     }
 
-    void transform_to_srgb(Image& image)
+    void transform_to_srgb(Image* image)
     {
-        const CanvasProperties& image_props = image.properties();
+        const CanvasProperties& image_props = image->properties();
 
         for (size_t ty = 0; ty < image_props.m_tile_count_y; ++ty)
         {
             for (size_t tx = 0; tx < image_props.m_tile_count_x; ++tx)
-                transform_to_srgb(image.tile(tx, ty));
+                transform_to_srgb(image->tile(tx, ty));
         }
     }
 
-    void write_png_image(
-        const bf::path&         file_path,
-        const Image&            image,
-        ImageAttributes         image_attributes)
+    /*
+     * Default Export Format
+     * OpenEXR   .exr          4-channel   16-bit (half)    Linear  
+     * TIFF      .tiff/.tif    4-channel   32-bit (float)   Linear
+     * RGBE      .hdr          3-channel   32-bit (float)   Linear
```

Comment:
Incorrect, RGBE is 8-bit integer + 8-bit shared integer exponent.

Reply:
Thanks, changed.